### ISSUE TITLE
A post with title via web= Great idea!

### ISSUE THREAD

**Issue by lfortanet** [e14n/pump.io#1128]
Hi everybody....I think ( like a newbie user of Pump.io ), would be interesting to have of the funcionality for to write a title when you're going to write a text via web app. 

One very important functionality on Pump.io are the titles and for that it's important to have that possibility over all variables and modes for use over it. 

Me, for example, use my work computer during a long time and I can´t to install a client app in that computer. The web app of Pump.io for me it's an important to use option. 

Good Job eveybody.

**Comment by strugee** [comment]
I'd like this to be behind a preference or a checkbox, so that there's not a huge title field displayed by default.

Personally I think that the fact that the web interface doesn't offer a title at the moment makes it feel like somewhat of a Twitter-like experience, as opposed to a blog-like experience, which I think is nice for the way the network is used.

To be clear: not saying it shouldn't be an option, because it _is_ useful. Just saying it shouldn't be displayed by default.

**Comment by h4ck3rm1k3** [comment]
I think that we could use microformats http://microformats.org/ to store the entire thing the body of the post and just declare the title and parts of the post with markup. The gui could display the markdown parsed as html with https://github.com/evilstreak/markdown-js. I have lots of things I would like to add in there, geojson/geo location. tags and other things could but put into the body.

**Comment by h4ck3rm1k3** [comment]
take a look at this for example http://www.leenau.com/2014/11/09/hreview-microformats-in-markdown/ just put the metadata at the top of the post data

**Comment by strugee** [comment]
@h4ck3rm1k3 I think you have some sort of misconception. Titles are already supported in Pump, they're just not exposed in the web interface. You need a native client to use them.

**Comment by h4ck3rm1k3** [comment]
@strugee thanks for clearing that up. and what about the microformats/other metadaTa?

**Comment by lfortanet** [comment]
@h4ck3rm1k3 Not always there is possibility to use a native client ....That´s the problem.

**Comment by strugee** [comment]
@lfortanet as I stated above, I'm not arguing against titles in the web interface, I'm arguing against the box being shown by default.

**Comment by strugee** [comment]
@h4ck3rm1k3 I'm not sure what kind of answer you're looking for here. Is there something wrong with ActivityStreams?

**Comment by h4ck3rm1k3** [comment]
@strugee Take a look at microformats and rdf. We can talk more about this later when I know more about pump.io.

**Comment by lfortanet** [comment]
@strugee Yes..I understand you. Thanks.

**Comment by evanp** [comment]
I favour this. I'd love to do a full-page editing interface for blog-like articles, with titles and so forth. I think short notes shouldn't have a title.

**Comment by strugee** [comment]
@evanp agreed. Except I was imagining that we'd have a checkbox somewhere in the note popup that said something along the lines of, "Give this note a title," and checking that would cause the title field to appear. Thoughts?

@h4ck3rm1k3 I'm already familiar with both of those technologies. But I don't understand where you think they should be used.

**Comment by h4ck3rm1k3** [comment]
I just think it would be great if we could embed things like geotags or
other structured data directly in the body of a post. Title is just another
metadata tag that could be added.

On Mon, Oct 26, 2015 at 11:10 PM, Alex Jordan <notifications@github.com>
wrote:

> @evanp <https://github.com/evanp> agreed. Except I was imagining that
> we'd have a checkbox somewhere in the note popup that said something along
> the lines of, "Give this note a title," and checking that would cause the
> title field to appear. Thoughts?
>
> @h4ck3rm1k3 <https://github.com/h4ck3rm1k3> I'm already familiar with
> both of those technologies. But I don't understand where you think they
> should be used.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e14n/pump.io/issues/1128#issuecomment-151358778>.
>



-- 
James Michael DuPont
Kansas Linux Fest http://kansaslinuxfest.us
Free/Libre Open Source and Open Knowledge Association of Kansas
http://openkansas.us
Member of Free Libre Open Source Software Kosova http://www.flossk.org
Saving Wikipedia(tm) articles from deletion http://SpeedyDeletion.wikia.com

**Comment by nachanon** [comment]
It bugs me somehow that there is a Title box in [picture posting dialog](https://cloud.githubusercontent.com/assets/5958794/14076842/88fa3a92-f50f-11e5-923a-5e41fe4edca0.png), but not in [normal posting dialog](https://cloud.githubusercontent.com/assets/5958794/14076738/d615ef34-f50e-11e5-8761-4e8ad1d6338e.png).

Since web interface does not offer any form of full-page editor, I think it's still useful to offer the option to set title of the post like one already offered on image posting; since setting up a client is not for everyone, especially not for new and casual user.

Disclaimer: I'm a new pump.io ([pumprock.net](https://pumprock.net/)) user, thus use web interface exclusively.

**Comment by strugee** [comment]
I'm generally in favor of doing this _somehow_, but it's not a priority right now

**Comment by jankusanagi** [comment]
Can we haz cheesetitlez? =)
